packages/react-auth/src/useHidePassInput.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export const useHidePassInput = (defaultValue = true) => {
```

Comment:
Once approved, remove this functionality from auth and implement it as a single component https://github.com/ttoss/ttoss/issues/99.